src/main/java/com/google/devtools/build/lib/authandtls/credentialhelper/CredentialHelperException.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.devtools.build.lib.authandtls.credentialhelper;
+
+/** An {@link Exception} thrown while invoking a credential helper. */
+public class CredentialHelperException extends RuntimeException {
```

Comment:
Can this extends `IOException`?

Reply:
Sure. I've also fixed the test assertions.